src/Microsoft.DotNet.Build.Tasks/PackageFiles/CloudTest.targets

```diff
@@ -0,0 +1,522 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Have not had time to read the whole PR yet, but it does make sense going to the CloudTestTasks project if it's still there.

However, if we're revving the version of buildtools, I'd much rather get rid of this file entirely.  CoreFX and CoreClr now use CloudTest.Helix.Targets and anyone adopting a new buildtools can bring their stuff up on the new Targets file in less than a day and have at least 90% fewer headaches when maintaining the file.